src/main/java/net/imglib2/img/display/imagej/ImageJVirtualStack.java

```diff
@@ -90,7 +90,7 @@ protected ImageJVirtualStack(final RandomAccessibleInterval< S > source, final C
 	protected ImageJVirtualStack(final RandomAccessibleInterval< S > source, final Converter< S, T > converter,
 			final T type, final int ijtype, ExecutorService service)
 	{
-		super( (int) source.dimension( 0 ), getDimension1Size( source ), null, null );
+		super( (int) source.dimension( 0 ), (int) source.dimension( 1 ), null, null );
```

Comment:
This is wrong. Now it will not work for 1D images anymore.

Reply:
It didn't work with 1d images before. The strategy for 1d images is to add an axis and the use ImageJVirtualStack.